src/gsy_e/models/strategy/load_hours.py

```diff
@@ -114,10 +119,9 @@ def state(self) -> LoadState:
         return self._state
 
     def _init_price_update(self, fit_to_limit, energy_rate_increase_per_update, update_interval,
-                           use_market_maker_rate, initial_buying_rate, final_buying_rate):
+                           initial_buying_rate, final_buying_rate):
 
         # Instantiate instance variables that should not be shared with child classes
-        self.use_market_maker_rate = use_market_maker_rate
```

Comment:
Have a look at the comment here. The initialisation of `self.use_market_maker_rate` seems to be here for a good reason.

Reply:
This could be right only in 1 case, if the extending class will override the `_init_price_update` method to implement some different behavior, however AFAIS there are no overriding methods of it

Reply:
I have to admit that I was too lazy to look deeper and find that out myself. Will do in the future though. Thanks a lot for explaining in such detail!